homeassistant/components/jlrincontrol/jlrpy.py

```diff
@@ -0,0 +1,517 @@
+""" Simple Python class to access the JLR Remote Car API
```

Comment:
This file shouldn't be in here as its not required. HomeAssistant downloads the required dependencies as required on bootup. Check the `requirements.txt` file IIRC.